src/index.js

```diff
@@ -1,6 +1,7 @@
 import { PluginSidebar } from "@wordpress/edit-post";
 import React, { render } from "@wordpress/element";
 import { registerPlugin } from "@wordpress/plugins";
+import domReady from '@wordpress/dom-ready';
```

Comment:
Should we add the dependency to `package.json`? 

Also, once the newer version of analytics is release, please update the `build/index.asset.php` file as well.

Reply:
We don't, @amartya-dev is aware of the update to `1.2.0` and updating the build, which is why we have still not merged this.